k2/python/csrc/torch/v2/doc/k2_ops.h

```diff
@@ -0,0 +1,151 @@
+/**
+ * @copyright
+ * Copyright      2021  Xiaomi Corp.  (authors: Wei Kang)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef K2_PYTHON_CSRC_TORCH_V2_DOC_K2_OPS_H_
+#define K2_PYTHON_CSRC_TORCH_V2_DOC_K2_OPS_H_
+
+namespace k2 {
+
+static constexpr const char *kTensorIndexSelectDoc = R"doc(
+Index a tensor with a 1-D tensor of indexes, alone dimension 0.
+
+Note:
+  Autograd is supported.
+
+Caution:
+  The index MUST have the dtype equals to `torch.int32` and
+  dimension equals to 1.
+
+>>> import torch
+>>> import k2
+>>> src = torch.tensor([0, 1, 2, 3, 4, 5, 6, 7, 8])
+>>> index = torch.tensor([2, 4, -1, 6, 1], dtype=torch.int32)
+>>> k2.index_select(src, index)
+tensor([2, 4, 0, 6, 1])
+>>> k2.index_select(src, index, 100)
+tensor([  2,   4, 100,   6,   1])
+>>> src = torch.tensor([[0, 1], [0, 2], [0, 3], [0, 4], [0, 5], [0, 6], [0, 7]])
+>>> k2.index_select(src, index)
+tensor([[0, 3],
+        [0, 5],
+        [0, 0],
+        [0, 7],
+        [0, 2]])
+>>> src = torch.tensor([0, 1, 2, 3, 4, 5], dtype=torch.float32, requires_grad=True)
+>>> index = torch.tensor([2, 4, -1, 1], dtype=torch.int32)
+>>> r = k2.index_select(src, index)
+>>> print (r)
+tensor([2., 4., 0., 1.], grad_fn=<IndexSelectFunction>>)
+>>> r.sum().backward()
+>>> print (src.grad)
```

Comment:
Please remove the space after `print`. The same goes for other `print` statements.